marimbabot_planning/scripts/dummy_client.py

```diff
@@ -21,19 +26,43 @@ def main_loop(self):
         """
         octave = int(input("Enter an octave (e.g. 4): "))
         while not rospy.is_shutdown():
-            input_str = input("Enter a sequence of hits (e.g. 'C, C#, A'): ")
+            # get input from user and create goal
+            input_str = input("Enter a sequence of hits (e.g. 'C, <C#, D>, C#, A'): ")
             goal = HitSequenceGoal()
             start_time = rospy.Time(0)
             duration = rospy.Duration(0.5)
-            for note in input_str.split(','):
+
+            # create regex for splitting the notes including only the notes from the scale (C, D, E, F, G, A, B)
+            # e.g. 'C, <C#, D>, C#, A' => ['C', '<C#, D>', 'C#', 'A']
+            # LIMITED TO A CHORD OF 2 NOTES !!!
+            regex_pattern = r'([CDEFGAB][#b]?|[<].*?[>])'
```

Comment:
I think you should make this regex more restricting.
Currently the inside part of the brackets is not checked and you can put in however many and any you symbols you want. I think this should be checked
```
>>> re.findall(regex_pattern, "C, <C#, D>, C#, A")
['C', '<C#, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, H, D>, C#, A")
['C#', '<C#, H, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, Hb, D>, C#, A")
['C#', '<C#, Hb, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, Hello good sir nice to see you, D>, C#, A")
['C#', '<C#, Hello good sir nice to see you, D>', 'C#', 'A']
```

Reply:
Sure, I mean the desired input is prompted and it the code is just for sending sequences without starting the behavior node, but I can do that.